build/docker/README.md

```diff
@@ -136,7 +135,14 @@ If you do not want to use the same scripts Travis CI does, you can do it manuall
 
 Build the image:
 
-    thrift$ docker build -t thrift build/docker/ubuntu-bionic
+Linux/Mac:
+
+    thrift$ docker build --build-arg uid=$(id -u) --build-arg gid=$(id -g) -t thrift build/docker/ubuntu-jammy
+
+Windows:
+
+    thrift$ docker build -t thrift build/docker/ubuntu-jammy
+
```

Comment:
Maybe tables on line 155 and 174 should be also updated with correct versions of core tools and compiler/languages?

Reply:
updated :+1:
would be nice to automate this in the future somehow